reconcile/terraform_resources.py

```diff
@@ -617,7 +617,7 @@ def run(
         defer(oc_map.cleanup)
 
     if print_to_file:
-        cleanup_and_exit()
+        cleanup_and_exit(tf)
```

Comment:
I also like this. 

It took me a bit of looking to convince myself that leaving the terrascript tmp files around when using --print-to-file definitely wasn't a deliberate choice for debugging. But I'm pretty certain such is not the case.